install/top-level-packages.rst

```diff
@@ -23,20 +24,40 @@ Example installation (:ref:`more info <newinstall-install>`):
 lsst\_distrib
 =============
 
-``lsst_distrib`` provides all the packages from ``lsst_apps`` along with these additional ones:
-
-.. https://github.com/lsst/lsst_distrib/blob/master/ups/lsst_distrib.table
+This package provides all of the core Science Pipelines functionality, together with additional measurement algorithms, support for a wider variety of instrumentation, and process execution middleware designed for running pipelines on a cluster.
+In addition to the contents of ``lsst_apps``, it provides the following packages:
 
+- `ap_verify <https://github.com/lsst/ap_verify>`_
+- `cbp <https://github.com/lsst/cbp>`_
+- `cp\_pipe <https://github.com/lsst/cp_pipe>`_
 - `ctrl\_execute <https://github.com/lsst/ctrl_execute>`_
+- `ctrl\_mpexec <https://github.com/lsst/ctrl_mpexec>`_
+- `ctrl\_orca <https://github.com/lsst/ctrl_orca>`_
 - `ctrl\_platform_lsstvc <https://github.com/lsst/ctrl_platform_lsstvc>`_
-- `datarel <https://github.com/lsst/datarel>`_
+- `ctrl\_pool <https://github.com/lsst/ctrl_pool>`_
+- `display\_firefly <https://github.com/lsst/display_firefly>`_
+- `display\_matplotlib <https://github.com/lsst/display_matplotlib>`_
+- `eigen <https://github.com/lsst/eigen>`_
```

Comment:
This is an implementation detail; eigen is provided via conda for everything else.  I would remove this line.

Reply:
Mm. I think I prefer leaving this in — it _does_ get installed, even if that's an implementation detail, and it seems potentially confusing to users not to mention it.

Reply:
I'm worried that people will think they don't have eigen if they don't install `lsst_distrib`.  Perhaps append "(only as a temporary workaround for jointcal; normally used from conda)"?